src/history/HistoryTests.cpp

```diff
@@ -204,6 +206,119 @@ TEST_CASE("History publish queueing", "[history][historydelay][historycatchup]")
           catchupSimulation.getApp().getLedgerManager().getLedgerNum());
 }
 
+TEST_CASE("History bucket verification",
+          "[history][bucketverification][batching]")
+{
+    /* Tests bucket verification stage of catchup. Assumes ledger chain
+     * verification was successful. **/
+
+    Config cfg(getTestConfig(1, Config::TESTDB_ON_DISK_SQLITE));
+    VirtualClock clock;
+    auto cg = std::make_shared<TmpDirHistoryConfigurator>();
+    cg->configure(cfg, true);
```

Comment:
why are you passing `true` here? Test app is not publishing to this archive, is it?

Reply:
Test app uses `PutRemoteFileWork` to upload buckets, which requires put command to be configured.